### ISSUE TITLE
Suggest better colors to the user in the color contrast UFIXIT

### ISSUE THREAD

**Issue by bagofarms** [ucfcdl/UDOIT#7]
When a color contrast issue is discovered, allow the user to select a color with better contrast based on the colors available in the Rich Content editor. These colors should be filtered down to those that provide enough contrast with the background color.

**Comment by bagofarms** [comment]
Work is already in progress for this one on the ColorPicker branch
